docs/network/governance.md

```diff
@@ -26,7 +26,7 @@ unilateral fashion.
 
 All parties are held accountable by legal agreements.
 
-## Node Checks and Balances
+## Nodes
```

Comment:
-1 to removing "Checks and Balances"... this entire page is about checks and balances. Suggest rewording to: "Checks and Balances for Nodes".

Reply:
Not all of the copy is about checks and balances. This page defines the role of the entity AND then describes how that entity is accountable to and can be checked by the others. The repetition of "checks and balances seems unnecessarily redundant to me.  A compromise might be the format. "Nodes, Checks and Balances".